pyvista/plotting/renderer.py

```diff
@@ -3193,7 +3191,7 @@ def add_legend(
             * ``'upper center'``
             * ``'center'``
 
-        face : str or pyvista.PolyData, optional
+        face : str | pyvista.PolyData, default: "triangle"
```

Comment:
Technically the valid input `None` is not covered by the current type line, but I don't have a concise way to fix this. Perhaps
```suggestion
        face : str | pyvista.PolyData | NoneType, default: "triangle"
```
Would that be better? I'm not sure.

Reply:
Works for me.